src/ForkCMS/Bundle/CoreBundle/Tests/Validator/UrlValidatorTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace ForkCMS\Bundle\CoreBundle\Tests\Validator;
+
+use ForkCMS\Bundle\CoreBundle\Validator\UrlValidator;
+use PHPUnit\Framework\TestCase;
+
+class UrlValidatorTest extends TestCase
+{
+    public function testExternalUrlValidation()
```

Comment:
Isn't this a bit generic. I would separate this into two test methods:
* testExternalUrlValidationWithValidUrls
* testExternalUrlValidationWithInvalidUrls

Reply:
fixed in a706fd97c